lib/pr.md

```diff
@@ -0,0 +1,46 @@
+<!-- 
```

Comment:
Please do not copy and/or modify the PR template. You just need to fill in it while creating a PR on github.